<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,0" columnstretch="1,1" columnminimumwidth="0,0">
    <item row="0" column="0">
     <widget class="QTabWidget" name="scriptTabs">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save Script</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run Script</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runSelectedButton">
        <property name="text">
         <string>Run Highlighted (F9)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="ConsoleWidget" name="consoleWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScripts">
    <property name="title">
     <string>Scripts</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad_Script">
     <property name="title">
      <string>Load Script</string>
     </property>
     <addaction name="actionFrom_Window"/>
     <addaction name="actionFrom_File"/>
    </widget>
    <addaction name="actionNew_Script"/>
    <addaction name="menuLoad_Script"/>
    <addaction name="actionSave_Script"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScripts"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Script">
   <property name="text">
    <string>New Script</string>
   </property>
  </action>
  <action name="actionSave_Script">
   <property name="text">
    <string>Save Script</string>
   </property>
  </action>
  <action name="actionFrom_Window">
   <property name="text">
    <string>From Window</string>
   </property>
  </action>
  <action name="actionFrom_File">
   <property name="text">
    <string>From File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.console</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
